pontoon/teams/templates/teams/includes/projects.html

```diff
@@ -1,22 +1,33 @@
 {% import 'projects/widgets/project_list.html' as ProjectList with context %}
 
+{% set locale_projects = locale.available_projects_list() %}
+{# Hide if team has any projects #}
+{% set hide_table = (locale.project_set.visible().count() == 0) %}
+{# Hide if there are any projects available for request #}
+{% set hide_request_toggle = projects.exclude(slug__in=locale_projects).count() == 0 %}
+
 <div class="projects items">
+  
+  {% if hide_table %}
+  <menu class="controls no-projects">
+    <div class="search-wrapper small clearfix hidden" >
+  {% else %}
   <menu class="controls">
     <div class="search-wrapper small clearfix">
+  {% endif %}
```

Comment:
Instead of this, you could do:
```html
<menu class="controls {% if hide_table %}no-projects{% endif %}">
  <div class="search-wrapper small clearfix" >
```

And then change the CSS rule to:
```css
.controls.no-projects > .search-wrapper {
  display: none;
}
```